pkg/policy/api/fqdn.go

```diff
@@ -26,11 +26,11 @@ import (
 
 var (
 	// allowedMatchNameChars tests that MatchName contains only valid DNS characters
-	allowedMatchNameChars = regexp.MustCompile("^[-a-zA-Z0-9.]+$")
+	allowedMatchNameChars = regexp.MustCompile("^[-a-zA-Z0-9_.]+$")
```

Comment:
The change itself looks reasonable, but I would usually expect such a change to be submitted against master first and then subsequently backport, so we can properly track the change as it goes through; otherwise it can be difficult to understand which PR is the "original" PR / source of truth and how to find out the backport status depending on the specific branches.

I understand that in some cases, the code differences on different branches can make this more difficult than in other cases, in which case maybe it's worth preparing a branch-specific version and performing review there. However I'd like this to be the rare exception rather than the rule, just to simplify the management of the changes. It's not such a big problem if these fixes are proposed occasionally, but if we have a wide range of core team members taking this approach, it becomes increasingly difficult to properly track the changes, decide which branches that we care to backport to, and make sure that we don't miss some branch or backport, get the fixes into the expected releases, etc.

The above approach to developing changes can be a little slower but if there are ways we can help to facilitate making this approach work better or more fluidly, I'd be interested to hear what you need to make this approach work. The basics are probably pretty well-known: Need the ability to test, need review, need the ability to build an image that can be shared with users to validate the changes. My ideal approach for this would be that we handle most of this against master, then for the image aspect maybe when we're happy with the testing/review/quality we can just backport & push a branch to a hotfix branch and kick off the image build from there.

For what it's worth, I believe that a PR is not necessary to generate an image based on the hotfix image workflow - I think we can just push a branch & trigger the process.

Reply:
> The change itself looks reasonable, but I would usually expect such a change to be submitted against master first and then subsequently backport, so we can properly track the change as it goes through; otherwise it can be difficult to understand which PR is the "original" PR / source of truth and how to find out the backport status depending on the specific branches.

TIL you do not need to open a PR for a hotfix build job :-)